lib/ecto/changeset.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Ecto.Changeset do
   @moduledoc ~S"""
   Changesets allow filtering, casting, validation and
-  definition of constraints when manipulating structs.
+  definition of constraints when manipulating structs that are defined via `Ecto.Schema`.
```

Comment:
Can we please remove this mention? As we also support regular structs as long as you give the types.